src/testMode/DemoMode.tsx

```diff
@@ -25,6 +25,10 @@ const ScreenRadioSelector = () => {
   const {forceScreen, setForceScreen} = useStorage();
   const screenData = [
     {displayName: 'None', value: 'None'},
+    {displayName: 'Not Exposed', value: 'NoExposureView'},
+    {displayName: 'Not Exposed (Region covered)', value: 'NoExposureCoveredRegionView'},
+    {displayName: 'Not Exposed (Region not covered)', value: 'NoExposureUncoveredRegionView'},
+    {displayName: 'Not Exposed (Region not set)', value: 'NoExposureNoRegionView'},
```

Comment:
It doesn't look like you're actually using these in the detox tests, is that correct? I would prefer if we don't add these so we don't have to expand the functionality of the test menu more than is necessary.